File: Selenium/Activities/Activity3.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/login-form")
    print("login page title is :"+driver.title)
    username=driver.find_element(By.XPATH,"//input[@id='username']").send_keys("admin")
    password=driver.find_element(By.XPATH,"//input[@id='password']").send_keys("password")
    driver.find_element(By.XPATH,"//button[@class='ui button']").click()
    message=driver.find_element(By.XPATH,"//div[@id='action-confirmation']").text
    print("Login confirmation message is :"+message)



    

File: Selenium/Activities/Activity3.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity3 {
    public static void main(String[] args) {
        WebDriver driver=new ChromeDriver();
        driver.manage().window().maximize();
        driver.get("https://v1.training-support.net/selenium/login-form");
        System.out.println("Login page title is  :"+driver.getTitle());
        // identifying username and entering user name
        driver.findElement(By.xpath("//input[@id='username']")).sendKeys("admin");
        // identifying password and entering Password
        driver.findElement(By.xpath("//input[@id='password']")).sendKeys("password");
        //Clicking on Sign in btn
        driver.findElement(By.xpath("//button[text()='Log in']")).click();
        //Print confirmation message
        String message=driver.findElement(By.xpath("//div[@id='action-confirmation']")).getText();
        System.out.println("Log in confirmation message is :"+message);
        driver.close();
    }
}
